packages/status-js/src/client/community/community.ts

```diff
@@ -3,20 +3,22 @@ import { SymDecoder } from 'js-waku/lib/waku_message/version_1'
 
 import { getDifferenceByKeys } from '../../helpers/get-difference-by-keys'
 import { getObjectsDifference } from '../../helpers/get-objects-difference'
-import { CommunityRequestToJoin } from '../../protos/communities'
-import { MessageType } from '../../protos/enums'
+import { ApplicationMetadataMessage_Type } from '../../protos/application-metadata-message_pb'
+import { CommunityRequestToJoin } from '../../protos/communities_pb'
+import { MessageType } from '../../protos/enums_pb'
 import { compressPublicKey } from '../../utils/compress-public-key'
 import { generateKeyFromPassword } from '../../utils/generate-key-from-password'
 import { getNextClock } from '../../utils/get-next-clock'
 import { idToContentTopic } from '../../utils/id-to-content-topic'
 import { Chat } from '../chat'
 import { Member } from '../member'
 
-import type {
-  CommunityChat,
-  CommunityDescription,
-} from '../../proto/communities/v1/communities'
+import type { CommunityDescription as CommunityDescriptionProto } from '../../protos/communities_pb'
+import type { CommunityChat } from '../chat'
 import type { Client } from '../client'
+import type { PlainMessage } from '@bufbuild/protobuf'
```

Comment:
⚠️ Uses `PlainMessage` utility type, because classes are passed around and `@bufbuild/protoc-gen-es` does not generate interfaces like `protons` does.

See ["OK, so why not generate both classes and interfaces?"](url).

Reply:
I must admit that I like how `protons` uses same name for both the interface and encoding function under a namespace, and thus allowing one import, by relying on type inference:

> the compiler infers whether it refers to the type alias or the namespace based on the usage. – https://stackoverflow.com/questions/62604227/namespace-with-the-same-name-as-type-alias-in-the-same-file

Reply:
I wonder why they didn't consider ES modules as a valid approach...

Reply:
They,

- say "ECMAScript module support" at https://www.npmjs.com/package/@bufbuild/protobuf
- have `module` set at https://github.com/bufbuild/protobuf-es/blob/main/packages/protobuf/package.json#L19
- build it at https://github.com/bufbuild/protobuf-es/blob/main/packages/protobuf/package.json#L16
- and export as such at https://github.com/bufbuild/protobuf-es/blob/main/packages/protobuf/package.json#L22

What am I missing 🤔?

Also, is there anything else before the PR can be approved?